lib/impress.sse.js

```diff
@@ -22,12 +22,11 @@ impress.sse.mixin = (application) => {
   application.sse.heartbeat = api.timers.setInterval(() => {
     let connectionId, connection;
     const connections = application.sse.connections;
-    for (connectionId in connections) {
-      connection = connections.get(connectionId);
+    connections.forEach((key, connection) => {
```

Comment:
The signature of `Map#forEach` is `(value, key, map)`, so the parameters of this callback must be swapped to `(connection, key)` which translates into just `(connection) => { ... }` since `key` isn't used here.